tools/convert_v1_to_v2.py

```
@@ -0,0 +1,172 @@
+                field = key[len(obj_type)+1:]
+                normalized_type = "requirement_mapping_set" if obj_type == "mapping" else obj_type
+                object_metadata.setdefault(normalized_type, {})[field] = val1
+
+    sheets_out = {
+        "library_meta": library_meta
+    }
+
+    used_tabs = set()
+
+    # --- Process each declared tab ---
+    for tab_name, obj_type, base_urn in tab_entries:
```

Ruff (0.8.2)

Loop control variable `base_urn` not used within loop body

(B007)